e2e/main.py

```diff
@@ -531,9 +581,20 @@ def call_bulk(bulks, *, batch_size, profiler=general_gas_profiler):
 
         burrows = list(range(1, 1001))
 
+        for burrow_id in burrows:
+            get_tez_tokens_and_make_checker_an_operator(checker, 100_000_000)
+
+        # TODO:
+        # call_bulk(
+        #     [
+        #         get_tez_tokens_and_make_checker_an_operator(checker, 100_000_000)
+        #         for burrow_id in burrows
+        #     ],
+        #     batch_size=350,
+        # )
         call_bulk(
             [
-                checker.create_burrow((burrow_id, None)).with_amount(100_000_000)
+                checker.create_burrow((burrow_id, None, 100_000_000))
                 for burrow_id in burrows
             ],
             batch_size=350,
```

Comment:
Just FYI - Based on the latest CI run, it looks like this batch size is too large (in terms of gas costs) now that we emit extra internal operations. I had this pretty tightly tuned before since it affects the total runtime of the test suite, but feel free to reduce the batch size as needed. https://github.com/tezos-checker/checker/runs/3869898775#step:6:7246

Reply:
PS: this might be an issue for any of the other lines which call `call_bulk()` if they include calls to entrypoints which now interact with the FA2 contract internally.

Reply:
Ah, thanks! :pray: I started looking into batching now, I really hope it's the last obstacle for CI.

Reply:
Yay, @dorranh I think I managed to get the e2e tests to pass! :smile:

Reply:
Huzzah!